features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/services/login/idp.jag

```diff
@@ -55,16 +55,24 @@
     var SystemApplicationDAO = Packages.org.wso2.carbon.apimgt.impl.dao.SystemApplicationDAO;
     var systemApplicationDAO = new SystemApplicationDAO();
     var clientId ;
-    systemApplicationDTO = systemApplicationDAO.getClientCredentialsForApplication(STORE_CLIENT_APP_NAME);
+    // this is to support migration from admin_store to admin_devportal
+    systemApplicationDTO = systemApplicationDAO.getClientCredentialsForApplication(STORE_CLIENT_APP_NAME_OLD);
```

Comment:
can't we rename the app here?

Reply:
it includes app name update in AM_SYSTEM_APPS table and rename in the identity server side as well.
Do we need to do it?

Reply:
As per the offline chat we decided to keep the migrated apps as it is.
Only new apps will be created as 'apim_devportal' and 'apim_publisher'